src/react/features/tools/update/updateTool.feature

```diff
@@ -10,41 +10,49 @@ Feature: Update tool
 
   Scenario: Update tool shown
     Then update tool is active
-    Then I see 'Update diory' in view
+    And I see 'Update diory' in view
+
+  # Done & cancel
+  Scenario: Done saves changes and closes the dialog
+    When I add ' some text' to text field
+    And I click Done button
+    Then I do not see 'Update diory' in view
+    And I see 'Diory 11 some text' in view
+
+  Scenario: Cancel cancels changes and closes the dialog
+    When I add ' some text' to text field
+    And I click Cancel button
+    Then I do not see 'Update diory' in view
+    And I see 'Diory 11' in view
 
   # Text
   Scenario: Update text
     # FIXME: Don't know how to select all & replace the text
     When I add ' some text' to text field
-    And click Done button
+    And I click Done button
     And I select tools button
     And I select update button
     And I take 'Diory 11' in focus
     Then I see 'Diory 11 some text' in text field
 
-  Scenario: Update text changes diory text
-    When I add ' some text' to text field
-    And click Done button
-    Then I see 'Diory 11 some text' in view
-
   # Image
   Scenario: Update image
     When I add '/test-image.png' to image field
-    And click Done button
+    And I click Done button
     And I select tools button
     And I select update button
     And I take 'Diory 11' in focus
     Then I see '/test-image.png' in image field
 
   Scenario: Update image changes background image
     When I add '/test-image.png' to image field
-    And click Done button
+    And I click Done button
     Then 'diory11' has 'linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url("http://localhost:3300/test-image.png")' as 'background-image'
```

Comment:
..includes '/test-image.png' is only relevant for the test.

Reply:
I'm not sure what you meant but it has to be a real image in order to Cypress show it correctly if look the video the video about the test.

Reply:
I mean that we should test only relevant things e.g. 'linear-gradient...' is not. In unit tests I would write expect('background-image').toInclude('/test-image-png'), not expect...toBe('linear-gradient...